src/client/pages/OfferNew/Introduction/Sidebar/index.tsx

```diff
@@ -193,8 +193,13 @@ export const Sidebar: React.FC<Props> = ({
       })
       .then(() => refetchAll())
   }
-  const hasMultipleQuotes = offerData.quotes.length > 1
 
+  const discounts: Array<React.ReactNode> = [
+    ...(isNorwegianBundle ? [textKeys.SIDEBAR_NO_BUNDLE_DISCOUNT_TEXT()] : []),
```

Comment:
Haha, I read it as **No Bundle Discount** at first :D

Reply:
Haha. Same here :D